abonagiri/abonagiri.md.txt

```diff
@@ -0,0 +1,14 @@
+# Abhinav Bonagiri (Abonagiri)
+## Profile Pic: 
+## Call me by this name: Abhinav
+## Cohort: July 2023
+## Email ID: Abhinavbonagiri@gmail.com
+## LinkedIn:
```

Comment:
I highly recommend this, if you explicitly do not want to create one then reply to this comment by saying, "I do not want to create my LinkedIn profile"